src/arrayMethodSort.js

```diff
@@ -1,11 +1,34 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareString = (a, b) => {
+    const A = String(a);
+    const B = String(b);
+
+    if (A > B) {
+      return 1;
+    }
+
+    if (A === B) {
+      return 0;
+    }
+
+    return -1;
+  };
+
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const current = this[i];
```

Comment:
You can use the array destructuring assignment to swap the elements. It is more concise and easier to read, e.g., [this[i], this[j]] = [this[j], this[i]].